source/jormungandr/tests/disruptions_tests.py

```diff
@@ -573,3 +613,52 @@ def test_line_reports(self):
             assert pt_object['embedded_type'] == 'stop_point'
             assert len(pt_object['stop_point']['links']) == 1
             assert pt_object['stop_point']['links'][0]['id'] == 'line_section_on_line_1'
+
+    def test_line_reports_with_current_datetime_outof_application_period(self):
```

Comment:
Would be nice to add a test with `_current_datetime=20170105T130000` > check that we get nothing.